Sources/GraphQLWebSocket/Client.swift

```diff
@@ -219,6 +219,8 @@ public class GraphQLWebSocket: WebSocketDelegate {
             self.close(code: closeCode)
             break
             
+        case .peerClosed:
+            self.close(code: 1000)
```

Comment:
Not sure if this is the right thing to do here, I'm not familiar enough sorry. Would love input here.

Reply:
> Not sure if this is the right thing to do here, I'm not familiar enough sorry. Would love input here.

```suggestion
            self.close(code: 1006)
```

I think the code should actually be `1006`